gmso/tests/test_views.py

```diff
@@ -0,0 +1,12 @@
+from gmso.core.views import PotentialFilters
+from gmso.tests.base_test import BaseTest
+
+
+class TestViews(BaseTest):
+    def test_view_atom_types_typed_ar_system(self, n_typed_ar_system):
+        atom_types = n_typed_ar_system().atom_types(repeat=True)
+        assert len(atom_types) == 100
+        atom_types_unique = n_typed_ar_system().atom_types(
```

Comment:
Add this same test, but with repeat=True and a filter. The filter should cover this. What if filter=False and a filter_by method is passed?